src/main/proto/ProgramService.proto

```diff
@@ -161,9 +161,9 @@ message ListUserResponse {
 }
 
 message UpdateUserRequest {
-  google.protobuf.StringValue user_id = 1;
-  UserRoleValue role = 4;
-  google.protobuf.StringValue short_name = 5;
+  google.protobuf.StringValue user_email= 1;
+  UserRoleValue role = 2;
+  google.protobuf.StringValue short_name = 3;
```

Comment:
As a general rule... we should not be changing the number used in the proto, only adding new fields with HIGHER numbers and deprecating current numbers.

However, no one is using this method yet so we have no worries of backwards compatibility issues so in this one case its nice to restart the count at 1.